docs/source/api.rst

```diff
@@ -45,7 +45,9 @@ This more advanced API is available in the `Dask distributed documentation
 <https://distributed.dask.org/en/latest/api.html>`_
 
 .. autofunction:: annotate
+.. autofunction:: get_annotations
 .. autofunction:: compute
+.. autofunction:: get_annotations
```

Comment:
```suggestion
```
I added it already 2 lines above

Reply:
Thank you. If you don't mind, let me know next time you're working on my PR at the same time as I do, it was unexpected.